### PR TITLE
Protonets cleaned

### PR DESCRIPTION
**Pull request by kris701**

### CODE REVIEW

File: P8/Data/SwedishLeaf/SwedishLeaf_TEST.tsv
Comment (jamadaha) [inline P8/Data/SwedishLeaf/SwedishLeaf_TEST.tsv:1]: Are we allowed to upload this? Shouldn't it be added to git ignore?

Comment (kris701) [reply P8/Data/SwedishLeaf/SwedishLeaf_TEST.tsv:1]: I guess thats true!
I have put the data folder in `.gitignore`, and added a little readme on how its intended to be used

**Review by jamadaha — APPROVED** [review]

File: P8/NetTrainers/ProtoNetTrainer/NetTrainer.py
Comment (NicholasHDK) [inline P8/NetTrainers/ProtoNetTrainer/NetTrainer.py:1]: hvad er grunden til at pakke det ind i en klasse?

Comment (NicholasHDK) [reply P8/NetTrainers/ProtoNetTrainer/NetTrainer.py:1]: Similarly for BaseNetTrainer

**Review by NicholasHDK — APPROVED** [review]

**Pull request merged**